### PR TITLE
Feature/tests

### LINKED ISSUE #32: There are no tests at all
image

(For the record the branch for tests is [feature/tests](https://github.com/FundersClub/eng-ops/tree/feature/tests))

### PR DESCRIPTION
**Pull request by j-friedman**
fixes #32

### CODE REVIEW

File: api/tests/test_handlers.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from django.test import TestCase
+
+from api.handlers import issue_handler
```
Comment (thomasrockhu) [inline api/tests/test_handlers.py]: `api` should come after a line break after `user_management`

File: api/tests/test_handlers.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+from issues.models import Issue
+from user_management.models import GithubUser
+
```
Comment (thomasrockhu) [inline api/tests/test_handlers.py:6]: two spaces

File: api/tests/test_tasks.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import json
+import mock
+
+from django.test import TestCase
+from django.utils import timezone
+
+from api.models import GithubRequest
```
Comment (thomasrockhu) [inline api/tests/test_tasks.py:11]: same here

File: api/views.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+from datetime import datetime
```
Comment (thomasrockhu) [inline api/views.py]: why?

File: .coveragerc [hunk 5]
```diff
@@ -0,0 +1,17 @@
+[run]
```
Comment (thomasrockhu) [inline .coveragerc:1]: @k I don't think coverage is setup properly on eng-ops, or rather, I don't see that coverage script running in github as it does in main app.  Is that expected?

File: api/tests/test_handlers.py (hunk 1 above)
Comment (j-friedman) [inline api/tests/test_handlers.py]: Why is that? Shouldn't it be in alphabetical order?

File: api/views.py (hunk 4 above)
Comment (j-friedman) [inline api/views.py]: It was leftover from the bad old days before `from django.utils import timezone` and `timezone.now()` I suspect.

File: api/tests/test_handlers.py (hunk 1 above)
Comment (thomasrockhu) [inline api/tests/test_handlers.py]: https://github.com/FundersClub/FundersClub/wiki/Style-Guide:-Python

`api` is the same module import

File: api/views.py (hunk 4 above)
Comment (thomasrockhu) [inline api/views.py]: should remove

**Comment by thomasrockhu** [comment]
LGTM barring the coverage comment

File: .coveragerc (hunk 5 above)
Comment (kbambridge) [inline .coveragerc:1]: @thomasrockhu Yes this was just to check coverage manually. What happens on the main app is circleci runs the tests with coverage and uploads the results to circlci.

**Pull request merged**